packages/proto-build/src/transformers.rs

```diff
@@ -0,0 +1,257 @@
+use std::collections::HashMap;
+use std::path::Path;
+
+use heck::ToSnakeCase;
+use heck::ToUpperCamelCase;
+use prost_types::{
+    DescriptorProto, EnumDescriptorProto, FileDescriptorSet, ServiceDescriptorProto,
+};
+use regex::Regex;
+use syn::__private::quote::__private::TokenStream as TokenStream2;
+use syn::{parse_quote, Attribute, Fields, Ident, Item, ItemEnum, ItemStruct, Type};
+
+use crate::{format_ident, quote};
+
+/// Regex substitutions to apply to the prost-generated output
+pub const REPLACEMENTS: &[(&str, &str)] = &[
+    // Use `tendermint-proto` proto definitions
+    ("(super::)+tendermint", "tendermint_proto"),
+    // Feature-gate gRPC client modules
+    (
+        "/// Generated client implementations.",
+        "/// Generated client implementations.\n\
+             #[cfg(feature = \"grpc\")]\n\
+             #[cfg_attr(docsrs, doc(cfg(feature = \"grpc\")))]",
+    ),
+    // Feature-gate gRPC client impls which use `tonic::transport`
+    (
+        "impl (.+)Client<tonic::transport::Channel>",
+        "#[cfg(feature = \"grpc-transport\")]\n    \
+             #[cfg_attr(docsrs, doc(cfg(feature = \"grpc-transport\")))]\n    \
+             impl ${1}Client<tonic::transport::Channel>",
+    ),
```

Comment:
```suggestion
    ),
    // Remove TryFrom since we are using rust 2021
    ("use std::convert::TryFrom", ""),
```
the `TryFrom` is already defined in `core::prelude::rust_2021::*` we can just remove this to surpress some compilcation warning.